src/pstl_offload.cpp

```diff
@@ -29,12 +29,54 @@
 namespace __pstl_offload
 {
 
-static auto
-__get_original_free()
+using __free_func_type = void (*)(void*);
+
+// list of objects for delayed releasing
+struct __delayed_free_list {
+    __delayed_free_list* _M_next;
+    void*                  _M_to_free;
```

Comment:
It seems like the indentation should be fixed. May be it is a GitHub fault since the code in "suggestion" has different number of whitespaces.

Reply:
Done.